### DISCUSSION TITLE
Inappropriate emoji auto-select

### DISCUSSION THREAD

**Discussion by erikfrid** [GitbookIO/community#387] [0. General bugs and questions]
Whenever I title a space with the name of our product "Middle" in the title, it auto-selects the middle finger emoji. This can easily be changed, but man it would have been terrible if I sent this to one of our customers!

<img width="275" alt="Screenshot 2023-11-01 at 12 48 26 PM" src="https://github.com/GitbookIO/community/assets/92760461/1c6cd132-fb83-4c84-829f-bf86f4b31d8e">

**Comment by eastside** [comment]
Would be great to fix!

Could Gitbook automatically filter out offensive emojis from being auto-selected?
